### ISSUE TITLE
Shortcut specific crashes

### ISSUE THREAD

**Issue by shuvashish76** [VishnuSanal/DialogMusicPlayer#14]
**Describe the bug**
Shortcut created using [AppManager](https://github.com/MuntashirAkon/AppManager) 🙈
![IMG_20220122_220016_838](https://user-images.githubusercontent.com/48226312/150647150-a19013d8-8a39-4143-8a07-6a949e239a69.jpg)

**To Reproduce**
1. Click on main activity shortcut
2. Either click on device "back" from device navigation-bar or click "Quit"

**Crash logs**
```
FATAL EXCEPTION: main
Process: phone.vishnu.dialogmusicplayer, PID: 8854
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.MediaPlayer.release()' on a null object reference
 at phone.vishnu.dialogmusicplayer.MainActivity.onBackPressed(:2)
 at android.app.Activity.onKeyUp(Activity.java:2994)
 at android.view.KeyEvent.dispatch(KeyEvent.java:2726)
 at h0.e.b(:3)
 at x.e.dispatchKeyEvent(Unknown Source:18)
 at d.h.dispatchKeyEvent(Unknown Source:10)
 at d.k$d.dispatchKeyEvent(:1)
 at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:354)
 at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:4747)
 at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4619)
 at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4161)
 at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4214)
 at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4180)
 at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4307)
 at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4188)
 at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4364)
 at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4161)
 at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4214)
 at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4180)
 at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4188)
 at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4161)
 at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4214)
 at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4180)
 at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4340)
 at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:4501)
 at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:2435)
 at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:1998)
 at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:1989)
 at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:2412)
 at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:141)
 at android.os.MessageQueue.nativePollOnce(Native Method)
 at android.os.MessageQueue.next(MessageQueue.java:325)
 at android.os.Looper.loop(Looper.java:142)
 at android.app.ActivityThread.main(ActivityThread.java:6518)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```
---
Another related crash

**To Reproduce**
1. Click on main activity shortcut
2. Click on "play"

**Crash logs**
```
FATAL EXCEPTION: main
Process: phone.vishnu.dialogmusicplayer, PID: 8689
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.media.MediaPlayer.isPlaying()' on a null object reference
 at z1.a.onClick(:1)
 at android.view.View.performClick(View.java:6294)
 at android.view.View$PerformClick.run(View.java:24774)
 at android.os.Handler.handleCallback(Handler.java:790)
 at android.os.Handler.dispatchMessage(Handler.java:99)
 at android.os.Looper.loop(Looper.java:164)
 at android.app.ActivityThread.main(ActivityThread.java:6518)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by VishnuSanal** [comment]
@shuvashish76 These crashes are expected! ;) Still, thanks for reporting - we can have a fix for these kinda bugs :)

**Comment by VishnuSanal** [comment]
@shuvashish76 these must be fixed with the latest release [v2.0.0] (I assume!) (https://github.com/VishnuSanal/DialogMusicPlayer/releases/tag/v2.0.0), please reopen if it is not the case :)
